eotsmanager/cmd/eotsd/daemon/keys.go

```diff
@@ -92,6 +94,26 @@ func saveKeyNameMapping(cmd *cobra.Command, keyName string) (*types.BIP340PubKey
 		return nil, fmt.Errorf("failed to load config: %w", err)
 	}
 
+	client, err := eotsclient.NewEOTSManagerGRpcClient(cfg.RPCListener)
+	if err == nil {
```

Comment:
I must say this seems like a lot guess work from the program it self. Isn't there a config option describing whether to used remote client or local db for key storage ?

Reply:
The thing here is if some other program has the db file open (`fpd start`) no other command can open the db to make modifications and it times out, so my idea here was check first if there is some client, updates by the client instead of trying to open the db file

Reply:
what I am hinting at is that this behaviour of guessing who have the db open maybe a bit suprising and may have some unintended consequences. (like what if somebody have some invalid `cfg.RPCListener` in config and will now import key to some unexpected eots manager)

In my mind it should be explicit choice of the operator whether to import key locally through db directly or through running client and:
- in case of using import to db directly and db being open by other process, return error.
- in case of using running client, and client being not available, return error

Reply:
understood, what we could do is add an flag option to send to a running client

Reply:
updated